### ISSUE TITLE
`isSubscribed()` always returns false before and after subscribing

### ISSUE THREAD

**Issue by its-kyle-yo** [super-ienien/twitter-webhooks#12]
Code: 
```js
    const { twitter } = config;
    const { accessToken, accessTokenSecret } = twitter;
    const user = {
      accessToken,
      accessTokenSecret,
      userId: "<User ID>"
    }
    let isSubbed = await WebhookService.userActivityHook.isSubscribed(user)
    console.log({ isSubbed });
    const sub = await WebhookService.userActivityHook.subscribe(user);
    isSubbed = await WebhookService.userActivityHook.isSubscribed(user)
    console.log({ isSubbed, sub });
```

Output: 

```
{ isSubbed: false }
[ { code: 355, message: 'Subscription already exists.' } ] // or
{
  isSubbed: false,
  sub: UserActivityEmitter {
    _events: [Object: null prototype] {},
    _eventsCount: 0,
    _maxListeners: undefined,
    id: '<UserID>'
  }
}
```
Not sure if my approach is correct but this seems to be case for the current pattern above.

**Comment by super-ienien** [comment]
Hi,

Thank you for reporting this. I will look into it as soon as possible.

**Comment by its-kyle-yo** [comment]
Any progress on this @super-ienien ? I looked into your codebase and `isSubscribed()` seems to be calling out to `account_activity/subscriptions/list.json` which does not seem to be a valid endpoint for Twitter any longer. 

Assuming it was due to the API change that happened within the last year.

**Comment by its-kyle-yo** [comment]
For those using this package I've opened PR #13 and forked the repo implementing the fix as well I will maintain my fork for any other breaking changes until maintenance has been resumed here. 

Just swap in your package.json
```
"twitter-webhooks": "git+https://github.com/its-kyle-yo/twitter-webhooks.git"
```

**Comment by super-ienien** [comment]
Fixed with #13
